mdrst.rst

```diff
@@ -2,15 +2,14 @@
 RST cheat sheet for Markdown fans
 #################################
 
-This is a comparison of Markdown and ReStructured Text to help people
-authoring or editing RST files.
+This cheatsheet provides a quick reference for translating
+Markdown to ReStructured Text (RST) to help someone author or edit
+RST files. 
 
 You can look at the `original Markdown file <md.md>`_ and the
 `original RST file <rst.rst>`_ that the examples were culled from
 to see the formatting as GitHub presents it.
 
-
```

Comment:
FYI: This happened when I added and then removed some text.  These lines don't need to be deleted.